content/applications/hr/recruitment.rst

```diff
@@ -318,99 +495,150 @@ process. To refuse an applicant, click on an applicant's card to navigate to a d
+:guilabel:`Filters` section. All applications for all job positions are presented in a list view,
+organized by stage.
 
-After refusal, the applicant's card is no longer visible in the job position's kanban view. To view
-refused applicants, navigate to the :menuselection:`Recruitment app --> Applications --> All
-Applications`. All applications for all job positions are presented in a list view. To view the
-refused applicants only, click :guilabel:`Filters`, then click :guilabel:`Archived/Refused`. All
-applicants that have been archived and/or refused appear in the list. To further organize this list
-and view refusals by job position, click :guilabel:`≣ Group By`, then click :guilabel:`Job`.
+.. _recruitment/new-refuse:
+
+Create or modify refuse reasons
+*******************************
+
+To view all currently configured refuse reasons, navigate to :menuselection:`Recruitment app -->
+Settings -> Applications: Refuse Reasons`. All the refuse reasons appear in a list view.
+
+To create a new refuse reason, click the :guilabel:`New` button in the top-left corner. A blank
+line appears at the bottom of the :guilabel:`Description` column.
+
+Type in the new refuse reason in the field. It is recommended to enter a reason that is short and
+concise, such as `offer expired` or `withdrew application`.
+
+Then, in the :guilabel:`Email Template` field, click on the field to reveal a drop-down menu.
+Select an :guilabel:`Email Template` from the list to be used when this refuse reason is selected.
+
+If a new :guilabel:`Email Template` is desired, type in the name for the new template in the field.
+Then, click :guilabel:`Create and edit...` and a :guilabel:`Create Email Template` form pop-up
+window appears.
+
+Enter a :guilabel:`Name` for the form and an email :guilabel:`Subject` in the corresponding fields.
+Enter the email content in the :guilabel:`Content` tab. Make any other desired modifications to the
+template, then click :guilabel:`Save & Close` to save the template and return to the
+:guilabel:`Refuse Reasons` list. The new template appears in the new refuse reason :guilabel:`Email
+Template` field.
+
+To make any modifications to a refuse reason, click on the line, and make any desired changes.
 
 New applicant
 ~~~~~~~~~~~~~
 
 An applicant card can be manually added if needed. If an applicant needs to be added to the list of
 prospective candidates and an applicant card has not been created yet (they have not applied for the
-job online yet) an applicant card can be easily added from the job position kanban view in one of
-two ways, using either the :ref:`Quick add <recruitment/quick-add-applicant>` icon or the
-:ref:`Create <recruitment/create-new-applicant>` button.
+job online) an applicant card can be easily added from the job position Kanban view in one of two
+ways, using either the :ref:`Quick add <recruitment/quick-add-applicant>` icon or the :ref:`New
+<recruitment/create-new-applicant>` button.
```

Comment:
There is no text on the UI stating "Quick add", so you can leave it lowercase and state it is a button, whereas the icon is the + sign
```suggestion
ways, using either the :ref:`quick add <recruitment/quick-add-applicant>` button or the :ref:`New
<recruitment/create-new-applicant>` button.
```